src/Domain/FeedbackGetDomain.php

```diff
@@ -2,13 +2,19 @@
 namespace Wheniwork\Feedback\Domain;
 
 use Predis\Client as RedisClient;
+use Wheniwork\Feedback\Service\HipChatService;
+use Wheniwork\Feedback\Service\GithubService;
 
 abstract class FeedbackGetDomain extends FeedbackDomain
 {
     protected $redis;
```

Comment:
I think this one can be private too, if the only access to redis is via `getRedisValue` and `setRedisValue`.